spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/metrics/CloudWatchProperties.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.autoconfigure.metrics;
+
+import java.net.URI;
+import org.springframework.boot.actuate.autoconfigure.metrics.export.properties.StepRegistryProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * {@link ConfigurationProperties} for configuring CloudWatch metrics export.
+ *
+ * @author Jon Schneider
+ * @author Dawid Kublik
+ * @author Bernardo Martins
+ * @author Eddú Meléndez
+ * @since 2.0.0
+ */
+@ConfigurationProperties(prefix = "management.metrics.export.cloudwatch")
+public class CloudWatchProperties extends StepRegistryProperties {
```

Comment:
For the sake of consistency, shouldn't it bind to `spring.cloud.aws.cloudwatch` and extends `AwsClientProperties`? Similar to how Stackdriver is done in GCP, where `StackdriverPropertiesConfigAdapter` adapts GCP properties to Micrometer.